src/releasers/node.ts

```diff
@@ -125,11 +125,14 @@ export class Node extends ReleasePR {
   // A releaser can implement this method to automatically detect
   // the release name when creating a GitHub release, for instance by returning
   // name in package.json, or setup.py.
-  static async lookupPackageName(gh: GitHub): Promise<string | undefined> {
+  static async lookupPackageName(
```

Comment:
it seems like [some code above (L71)](https://github.com/googleapis/release-please/blob/9ba60ee3f8f3bfc9c0dee3180dce3b16386aa1b1/src/releasers/node.ts#L71) could be DRYed out using this method unless I'm missing something?

Reply:
nm, I see you need `contents` further down so L71 should probably just stay as is.